google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -812,7 +812,10 @@ private void appendLoop() {
         this.streamConnection.send(originalRequestBuilder.build());
       }
     }
+    cleanup();
+  }
 
+  private void cleanup() {
```

Comment:
name this shutdownAndCleanup?

Reply:
I renamed it to cleanupConnectionAndRequests().